modules/apps/transfer/keeper/relay_test.go

```diff
@@ -885,7 +885,7 @@ func (suite *KeeperTestSuite) TestOnAcknowledgementPacketSetsTotalEscrowAmountFo
 func (suite *KeeperTestSuite) TestOnTimeoutPacket() {
```

Comment:
The changes in this file are unrelated to the main purpose of the PR to increase coverage of `OnTtimeoutPacket`, right? We would normally do this in a separate PR, so that reviews are easier; or at least would add a comment with the purpose of the changes to help the reviewers.

Reply:
Good point, this was somewhat orthogonal to the original issue. It was just a case I saw that was not covered when going through the coverage report. I can easily pull this out into a separate PR. Would you prefer an issue for it, a comment (I am adding that in any case now) or a PR without an issue?

Reply:
A separate PR would be fine by me. Thank you, @gjermundgaraba!

Reply:
Done: https://github.com/cosmos/ibc-go/pull/6460